### ISSUE TITLE
Promotion action and rule validation error

### ISSUE THREAD

**Issue by ahmadrabie** [Sylius/Sylius#4332]
@pamil and @pjedrzejewski In your online demo all of `sylius_entity_to_identifier` field types have validation error.
![screenshot from 2016-02-27 12 38 53](https://cloud.githubusercontent.com/assets/10043763/13372211/de91184e-dd50-11e5-82aa-530184b95a4e.png)

Input of ResourceBundle `ObjectToIdentifierTransformer->reverseTransform($value)` is not Identifier value, it is object.
![screenshot from 2016-02-27 12 55 30](https://cloud.githubusercontent.com/assets/10043763/13372240/973f6058-dd51-11e5-9773-f7c1a06e1722.png)

so validation constraint return false:
```php
                'constraints' => [
                    new Type(['type' => 'numeric']),
                ],
```
because it inspect to receive a number but the value is object? 
Can you help me to fix this issues?

**Comment by flajos** [comment]
It's solved in fc345d6.

**Comment by michalmarcinkowski** [comment]
👍
